pkg/db/factory.go

```diff
@@ -32,7 +33,7 @@ func NewConnectionFactory(configFile string, migrate bool, debug bool) (Connecti
 	case "sqlite":
 		connFact, err := createSqliteConnectionFactory(encKey, debug, blockQueries, logQueries)
 		if err != nil {
-			return nil, err
+			return nil, errors.Wrap(err, "error creating sqliteConnectionFactory")
```

Comment:
If this is the only place need to use `errors` package, it's not necessary to introduce it, and be careful when introducing external dependencies, the more introduced, the more compatibility issue has to concern later:)

If you check the disclaimer of this package, after Go2 is released, it's into maintenance mode

https://github.com/pkg/errors
With the upcoming Go2 error proposals this package is moving into maintenance mode. The roadmap for a 1.0 release is as follows:

    0.9. Remove pre Go 1.9 and Go 1.10 support, address outstanding pull requests (if possible)
    1.0. Final release.

Reply:
use  `fmt/errors` should be enough in this case.

Reply:
I used it, since we use it everywhere in Kyma/reconciler project. Thus, i am not introducing a new dependency to our project here.
I would for now stick with the our default way of doing, and then change it when we change it everywhere